### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by devbaraus**
Just a simple update on the readme. 
It took me a while to get started even with the old example, the result I expected (the one in the image) wasn't the same I had. 
Also, I added some more configuration like the visConfig so other people just from the read know they can change more stuff in the visualization.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -158,27 +158,40 @@ And define our draw() function:
...
                     label: "name",
                     value: "pagerank",
                     group: "community",
                     [NeoVis.NEOVIS_ADVANCED_CONFIG]: {
                         function: {
-                            title: (node) => viz.nodeToHtml(node, [
-                                "name",
-                                "pagerank"
-                            ])
+                            title: (node) => node.properties.name
```
Comment (thebestnom) [inline README.md:178]: This is wrong, the point of the title is to be popup with all the data same for the relationship

File: README.md [hunk 2]
```diff
@@ -158,27 +158,40 @@ And define our draw() function:
...
                     }
                 }
             },
             relationships: {
                 INTERACTS: {
                     value: "weight"
+                    [NeoVis.NEOVIS_ADVANCED_CONFIG]: {
+                        function: {
+                            title: (relationship) => relationship.properties.weight,
+                            label: (relationship) => 'INTERACTS',
```
Comment (thebestnom) [inline README.md:189]: This is a good place to use static and not function 😅

**Review by thebestnom — CHANGES_REQUESTED** [review]
Mostly good changes, thanks!

File: README.md (hunk 1 above)
Comment (devbaraus) [reply README.md:178]: When I ran the version with viz.nodeToHtml it throws an error. Is it just for me? Could you check it?

File: README.md (hunk 2 above)
Comment (devbaraus) [reply README.md:189]: True :sweat_smile:

File: README.md (hunk 1 above)
Comment (thebestnom) [reply README.md:178]: Cam you send here traceback?

Comment (devbaraus) [reply README.md:178]:
```
index.html:62 Uncaught (in promise) TypeError: viz.nodeToHtml is not a function
    at title (index.html:62:51)
    at RB.BB (neovis.js@2.1.0:2:1153480)
    at RB.ZB (neovis.js@2.1.0:2:1154335)
    at ZB.next (<anonymous>)
    at Function.all (<anonymous>)
    at RB.UB (neovis.js@2.1.0:2:1154913)
    at async RB.VB (neovis.js@2.1.0:2:1155187)
    at async neovis.js@2.1.0:2:1156717
    at async Promise.all (index 0)
```

Comment (devbaraus) [reply README.md:178]: Also, if the person is using react he won't find this function in the package (my case).

Comment (thebestnom) [reply README.md:178]: Yeah, it should be NeoVis.objectToHtml (on the class, not on the instance (make sure you are on latest as on 2.0.2 it was imported separatly)

Comment (devbaraus) [reply README.md:178]: I'm using version 2.1.0. Still

```
index.html:62  Uncaught (in promise) TypeError: NeoVis.nodeToHtml is not a function
    at title (index.html:62:54)
    at RB.BB (neovis.js@2.1.0:2:1153480)
    at RB.ZB (neovis.js@2.1.0:2:1154335)
    at ZB.next (<anonymous>)
    at Function.all (<anonymous>)
    at RB.UB (neovis.js@2.1.0:2:1154913)
    at async RB.VB (neovis.js@2.1.0:2:1155187)
    at async neovis.js@2.1.0:2:1156717
    at async Promise.all (index 0)
title @ index.html:62
BB @ neovis.js@2.1.0:2
ZB @ neovis.js@2.1.0:2
UB @ neovis.js@2.1.0:2
```

Comment (thebestnom) [reply README.md:178]: How are you importing neovis?

Comment (devbaraus) [reply README.md:178]: How should I import? This error is on a pure html file...

```
<div id="viz"></div>
<script src="https://unpkg.com/neovis.js@2.1.0"></script>

<script type="text/javascript">
    window.onload = () => {
        draw();
    }

    let neoViz;

    // https://visjs.github.io/vis-network/docs/network
    function draw() {
        const config = {
            containerId: "viz",
            neo4j: {
                serverUrl: "bolt://localhost:7687",
                serverUser: "neo4j",
                serverPassword: "abacaxi1234",
            },
            visConfig: {
                nodes: {
                    shape: 'circularImage', //dot, circle, image, circularImage
                },
                edges: {
                    arrows: {
                        to: {enabled: true}
                    }
                },
            },
            labels: {
                Character: {
                    label: "name",
                    value: "pagerank",
                    group: "community",
                    [NeoVis.NEOVIS_ADVANCED_CONFIG]: {
                        function: {
                             title: (node) => NeoVis.nodeToHtml(node, [
                                "name",
                                "pagerank"
                            ]),
                            image: (node) => 'https://th.bing.com/th/id/R.daa762238acb5391d760f15512430c93?rik=1RaEtbM%2fwS4y5w&riu=http%3a%2f%2fantoniocorreia.smugmug.com%2fphotos%2f398317216_WxpWA-M.jpg&ehk=Lm4s5VQYoT7GySHeYukp6krp5hjYjZsW%2bMgngedUpbA%3d&risl=&pid=ImgRaw&r=0',
                            // label: (node) => null,
                        }
                    }
                }
            },
            relationships: {
                INTERACTS: {
                    value: "weight"
                }
            },
            initialCypher: "MATCH (n)-[r:INTERACTS]->(m) RETURN n,r,m LIMIT 100"
        };

        neoViz = new NeoVis.default(config);
        neoViz.render();
    }
</script>
```

Comment (thebestnom) [reply README.md:178]: NeoVis.objectToTitleHtml

Comment (thebestnom) [reply README.md:178]: I renamed it in 2.x and forgot 😅

File: README.md [hunk 3]
```diff
@@ -176,9 +186,15 @@ And define our draw() function:
             relationships: {
                 INTERACTS: {
                     value: "weight"
+                    [NeoVis.NEOVIS_ADVANCED_CONFIG]: {
+                        function: {
+                            title: (relationship) => relationship.properties.weight,
```
Comment (thebestnom) [inline README.md:191]: Also here you are using title wrong 😅

**Review by thebestnom — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (devbaraus) [reply README.md:191]: How would it be?

Comment (thebestnom) [reply README.md:191]: Same as the title function in nodes 😅

Comment (thebestnom) [reply README.md:191]: https://github.com/neo4j-contrib/neovis.js/pull/346#discussion_r1197142214
I already commented here, my bad for not commenting on both lines 😅

**Pull request closed without merging**